pyaedt/icepak.py

```diff
@@ -3902,27 +3912,37 @@ def assign_hollow_block(
         object_name : str
             Name of the object.
         assignment_type : str
-            Type of the boundary assignment. The accepted types are: "Total Power", "Heat Flux",
-            "Temperature" or "Heat Transfer Coefficient".
+            Type of the boundary assignment. Options are ``"Heat Transfer Coefficient"``,
+            ``"Heat Flux"``, ``"Temperature"``, and ``"Total Power"``.
         assignment_value : str or dict
-            String with value and units of the assignment. If ``"Total Power"`` is assignment_type,
-            ``"Joule Heating"`` can be used.
-            Also, a dictionary can be used for temperature dependent or transient assignment.
-            The dictionary should contain three keys: ``"Type"``, ``"Function"`` and
-            ``"Values"``. Accepted ``"Type"`` values are: ``"Temp Dep"`` and ``"Transient"``.
-            Accepted ``"Function"`` are: ``"Linear"``, ``"Power Law"``, ``"Exponential"``,
-            ``"Sinusoidal"``, ``"Square Wave"`` and ``"Piecewise Linear"``. ``"Temp Dep"`` only
-            support the latter. ``"Values"`` contains a list of strings containing the parameters
-            required by the ``"Function"`` selection (e.g. ``"Linear"`` requires two parameters:
-            the value of the variable at t=0 and the slope of the line). The parameters required by
-            each ``Function`` option is in Icepak documentation. The parameters must contain the
-            units where needed.
+            String with value and units of the assignment. If ``"Total Power"`` is
+            assignment_type, ``"Joule Heating"`` can be used.
+            For a temperature-dependent or transient assignment, a dictionary can be used.
+            The dictionary should contain three keys:
+            ``"Type"``, ``"Function"``, and ``"Values"``.
+            - For the ``"Type"`` key, accepted values are ``"Temp Dep"`` and ``"Transient"``.
```

Comment:
Again, for proper doc rending, we need to have a blank list before the first list item and after the last list item.